firefox-ios/Client/Frontend/Browser/Toolbars/Redux/ToolbarMiddleware.swift

```diff
@@ -604,6 +612,24 @@ class ToolbarMiddleware: FeatureFlaggable {
         return navToolbarModel
     }
 
+    private func getMiddleButtonAction(url: URL?, isPrivateMode: Bool) -> ToolbarActionState {
+        let isFeltPrivacyUIEnabled = featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly)
+        let isFeltPrivacyDeletionEnabled = featureFlags.isFeatureEnabled(.feltPrivacyFeltDeletion, checking: .buildOnly)
+        let isToolbarRefactorEnabled = featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly)
```

Comment:
We don't need to check if toolbar feature is enabled as this code (middleware) is already behind that feature flag.